app/views/user_mailer/delete_work_notification.text.erb

```diff
@@ -1,9 +1,12 @@
 <% content_for :message do %>
-Dear <%= @user.default_pseud.byline %>,
-  
-Your work "<%= @work.title.html_safe %>" was deleted at <%= (@work.pseuds.count > 1 && @user != User.current_user) ? "the request of " + text_pseud(User.current_user.default_pseud) : "your request" %>.
-  
-If you have questions, please contact Support at <%= root_url %>support.
-  
-Attached is a copy of your work for your reference.
-<% end %>
\ No newline at end of file
+<%= t('.hello', user:  @user.default_pseud.byline) %>
+<%= (@work.pseuds.count > 1 && @user != User.current_user) ?
+t('.part1_other',
+        title: @work.title,
+        pseud: User.current_user.default_pseud) :
+t('.part1_yourself',
+        title: @work.title) %>
+<%= t('.part2', support: t('.support')+' '+new_feedback_report_url ) %>
```

Comment:
That is correct sarken. It looks like:

If you have questions, please contact Support http://archiveofourown.org/support.
Attached is a copy of your work for your reference.

And html

<p>If you have questions, please <a href="http://archiveofourown.org/support" style="color:#990000">contact Support</a>.</p>
<p>Attached is a copy of your work for your reference.</p>